vhost-user-backend/src/event_loop.rs

```diff
@@ -57,6 +58,44 @@ impl std::error::Error for VringPollError {}
 /// Result of vring epoll operations.
 pub type VringEpollResult<T> = std::result::Result<T, VringPollError>;
 
+bitflags! {
+    #[derive(Debug, PartialEq, PartialOrd)]
+    pub struct VringPollEvent: u32 {
```

Comment:
`VringEpollResult` is too verbose, is a type that will be part of the public API of `VringEpollHandler`, so maybe something like:
- `EventType`
- `Interest`

are better